chicken-sloths/bangazon-site-sprint3
# Fixes #1 Cancel Order

# Description
A user can cancel an order. This removes all relationships with order on orderProducts and removes the open order from customer

## Related Ticket(s)
Fixes #1  

## Problem to Solve
If the user changes their mind about an order, they can cancel it


## Expected Behavior
Cancellation of an order and redirect to cart

## Steps to Test Solution

1.  ``` npm start ```
1.  log in
1. add items to your cart
1. go to your cart
1. check pgadmin to see the order exists and so do the relationships 
1. click cancel and check pgadmin again to see they have been removed
